frontend/src/components/utils/table/sorting.ts

```diff
@@ -30,14 +30,34 @@ export const cycleSorting = (current?: Sorting) =>
 
 /**
  * Build an URL sorting parameter from a `sorting` object. The function iterates through the object's keys and returns
- * the sorting parameter corresponding to the _first_ key containing either `Sorting.Ascending` or `Sorting.Descending`
- * (in other words, multi-sort is _not_ supported).
+ * the value of the sorting parameter corresponding to the _first_ key containing either `Sorting.Ascending` or
+ * `Sorting.Descending` (in other words, multi-sort is _not_ supported).
+ *
+ * For instance:
+ * ```
+ * sortingToApiParameter(
+ *     {
+ *         myPiche: Sorting.Descending,
+ *         myPoche: Sorting.Ascending,
+ *     },
+ *     {
+ *         myPiche: my_piche
```

Comment:
Who is `my_piche` here ?

Reply:
See `mapping`: “an optional object which maps the key used by `sorting` to the keys used by the backend. Useful for
 * converting camelCase into snake_case.”

Reply:
Ok, in that case maybe we could automatically convert from snake case to camel case?